zipkin-cassandra/src/test/scala/com/twitter/zipkin/storage/cassandra/CassandraSpanStoreSpec.scala

```diff
@@ -36,4 +40,19 @@ class CassandraSpanStoreSpec extends SpanStoreSpec {
   override lazy val store = new CassandraSpanStore(new Repository(keyspace, cluster))
 
   override def clear = cluster.connect().execute("DROP KEYSPACE IF EXISTS " + keyspace)
+
+  override def setTimeToLive() {
+    ready(store(Seq(span1)))
+    ready(store.setTimeToLive(span1.traceId, 1234.seconds))
+
+    for( i <- 1 to 100) {
+      // Repository.storeXXX() methods write asynchronously but don't return a Future so we can't reliably test.
+      //  just wait and loop instead
+      java.lang.Thread.sleep(50)
+      if ((result(store.getTimeToLive(span1.traceId)) - 1234.seconds).abs.inMilliseconds <= 10) {
```

Comment:
fixed with https://github.com/thelastpickle/zipkin/commit/b518cbbf69dfb97f68328b9e05d5388b66f7f057

which has been squashed into this PR